serial.c

```diff
@@ -300,6 +300,10 @@ int read_comport(char *response)
    bzero(tmp,64);
 
    FD_ZERO(&readfs);
+   if (fdtty < 0) {
```

Comment:
Sorry, the github UI is weird. *this* commit is part of another PR and doesn't belong in this one.